backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/postalcode/PostalCodeProcessor.java

```diff
@@ -0,0 +1,40 @@
+package com.bakdata.conquery.models.datasets.concepts.filters.postalcode;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.univocity.parsers.common.DataProcessingException;
+import com.univocity.parsers.common.ParsingContext;
+import com.univocity.parsers.common.processor.AbstractRowProcessor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+public class PostalCodeProcessor extends AbstractRowProcessor {
+	final private List<Double> data = new ArrayList<>();
+	private int recordsNumber = 0;
+	final private List<String[]> notCorrectRows = new ArrayList<>();
+
+
+	@Override
+	public void processStarted(ParsingContext context) {
+		super.processStarted(context);
+		context.skipLines(1); //skip headers
+	}
+
+	@Override
+	public void rowProcessed(String[] row, ParsingContext context) {
+		try {
```

Comment:
Jetzt summierst du alle spalten einer Zeile auf, was erreicht das? Du willst doch am Ende Daten in der Form haben `Table<Plz1,Plz2,Distance>`. Es muss nicht zwingend eine Table sein, aber das einsammeln solltest du schon hier machen. Das ist dann nämlich relativ schnell.

Reply:
Also, du möchtest ja zum einlesen wissen ob du ein paar schonmal gesehen hast, wenn ja ignorierst du es. Wenn nein fügst du es einer Liste ein. Vielleicht könntest du 2 Datensttrukturen benutzen, einmal ein `Set<Pair<int, int>>` was dir überprüft ob du die Einträge schon kennst, und eine `List<Record<plz1,plz2,distance>>` (Das ist nur eine Skizze wie das auszusehen hat). Am Ende sind wir ausschließlich an der Liste der Records interessiert, um über sie drüber zu iterieren

Reply:
Ihr hattet gemeint, da es besser ist, mit Listen zu arbeiten und Table sollte ich vermeiden oder, weil die Itterationen dadurch schneller wären oder? und   laut eurer Aussage sollte der Set vermieden werden.

Reply:
Genau am Ende soll eine Liste bei rauskommen, wie du die erzeugst ist nicht so wichtig, da kannst du zum Beispiel die Table benutzen, oder eben ein Set zum tracken der schon gesammelten Daten.